### DISCUSSION TITLE
The Doppelganger Problem and Music

### DISCUSSION THREAD

**Discussion by impotentfield** [enzo1982/freac#365] [General Discussion]
I wanted to start a general topic on my current problem with living in the United States. People carry on certain prime aspects of their lives into their current application of mental confluence (from living in the actual homes in the states) at the podium of the internet communications ... sitting area.... :|

in my life i find some things similar and some things not when it comes to people on the internet and in real life. when i write in my application of actual living and dwelling inside my fitness in education all kinds of things for me can be achieved. when noticing at this point i havent been up to making many friends in my REAL life and my internet life seems to make me up for my grasp on my communcation is placed in a sort of more efficient reality. if at all you dont see many things but shortcuts to the human relation and find things helpful in respect to where education can be in tune with fleunt mental allowance for breathing the moment of reality into the fine and not falsely maintaining life of yourself and others there is something i want to make some concepts aware just to make sure i have a good idea going where there might be some feedback. here is my current thought on doppleganger in my blog entry (recent) and i thought it might help with a sort of lucid placement of attention on the subject where people dont really have a clue about probably and they need a prime adjustment.

https://composuedol.blogspot.com

^-- this blog is about doppelganger

Update:
It seems when dealing with the public interest in aspects where, takeing in moments from internal stimulation from whereabout you at least get - first - moments of stimulation and second moments of stimulation you get all sorts of ideals to take on whereas if your engaging with the release of information from namely education in prime examples in which you take your own information coming from you or the other way of information going on in feedback where the information is coming from other people from 'other' sources you get the concept of any one main point exposing coefficience in the complex information where people take considerable attention to and make attention towards expression. in music and literature people more liekly get their complex from the composition of the observations, but they are not feeding to anyone. How can people survive a world without context where the prime composition of the context doesnt really give any internal reference to probable supports from inside the maintainment of certain things going on where the formal expression was to go out to others and give sustainment where the context was producing good intelligent feelings where in formal rights to the mind the expressions give oxygen to the mind and give 'good' feelings to it also and therefore good feelings work in circulation.... in the complex of disease whereas doppelganger is to manifold its self in maternal confidence in the negative aspects, the main point is to remove the problem because good communication goes well with the good feelings and good minds dont want bad communcation. its confusing that good information doesnt produce anything, because the main problem is doppelganger, its true....

**Comment by Lee-Carre** [comment]
That was something of a challenge to even parse, then follow & comprehend. \(Paragraph\|sentence\)-breaks (more of them) would help. I'm failing to see the relevance to music (or audio in general), at all (I tried).

What, exactly, do you mean by ‘the doppelgänger’, in this context?

I get the impression that you're relatively young, so I'll offer some advice from several decades of experience. I mean it kindly.

It sounds, to me, like you're an uncommon character, surrounded by normies, without those you truly consider to be peers, and struggling as a result, with no-one to even have a decent conversation with. If so: my sympathies, I relate.

I'm reminded of the advice of Aaron Clarey, especially in his book *The Curse of the High IQ*, in which he recommends as a solution to the perpetual problem of being surrounded by mediocre normies, is to find peers via the InterWeb; because (not his words) the network evaporates geography. So, while you might be a rarity in your local community, when you have access to a global network then distance no longer matters, and others who are also local rarities end up being somewhat numerous (even if still relatively rare). The InterWeb also enables all sorts of very niche communities. Plus, the smart people are running the infrastructure, in that context, which changes things some, to meat-space.

You mention communication, especially modes/style, several times.
InterWeb comms (having grown up on dial-up (and pre-56Kbps dial-up, at that), and so accustomed to text-only channels) are, in my experience, inherently different to in-person communication. Having thought about it, I don't regard the likes of (especially) video or (somewhat) audio as inherently superior to textual. Text channels have their advantages:
* asynchronous
* non-realtime
* possible to have multiple (mostly-)simultaneous conversations (in the same sense as your CPU is multi-tasking via context-switching), usually with multiple people, but sometimes with the same person (so long as you're each quoting what you're replying to, in each message), and occasionally both of those combined
* text is the most accessible format (in so many ways)
* covversation logs are searchable (eg grep) and easy to compress & archive efficiently. Decades of conversation needs only megabytes, rather than terabytes as with video or even audio
* I would argue that being forced (due to technical limitations) to use a text-only channel, encourages one to better articulate themselves, since the various cues (eg gestures, facial expression) used in meat-space comms, are irrelevant in text, and must be expressed via better-crafted words, rather than relying on the listener deducing one's true meaning from context and other clues
* voice channels have *some* advantages to text, but the two are simply different, rather than one being inherently better
* video adds very little, for conversations which are impersonal; for conversations with those whom you share some kind of intimacy, video may have some benefits

I'd encourage you to not be so concerned about what others think, or busy themselves with. Find what you're \(talented at\|passionate about\), and excel. In the long-run, that'll more likely lead to a sense of satisfaction and meaning, than endless amounts of chin-wagging with normies.

If you're in a small town, then perhaps you should consider escaping to somewhere larger, with more niche social groups.
